python/ray/tests/test_basic_3.py

```diff
@@ -169,6 +171,7 @@ def f():
         wait_for_pid_to_exit(pid)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Hangs on windows")
```

Comment:
It has a performance issue. I recently bumped its time limit to 510s. Once @wuisawesome completes his PR on scheduling, then I will pick its investigation further. Please let me know if that's okay. @wuisawesome

Reply:
yup, i'll reenable it (actually probably can enable it now)

Reply:
Okay. If it works then that would be great. Otherwise I am willing to investigate the causes.